pkg/controller/workload/job/job_controller.go

```diff
@@ -373,6 +443,23 @@ func (r *JobReconciler) ensureAtMostOneWorkload(ctx context.Context, job *batchv
 	// Find a matching workload first if there is one.
 	var toDelete []*kueue.Workload
 	var match *kueue.Workload
+
+	if parentWorkload(job) != "" {
```

Comment:
better to define `pwName := parentWorkload(job)` and use `pwName` in other places

Reply:
Done